src/ShallowTraversal.js

```diff
@@ -151,3 +151,23 @@ export function getTextFromNode(node) {
     .join('')
     .replace(/\s+/, ' ');
 }
+
+export const isAReactComponentNode = node => typeof node.type === 'function';
```

Comment:
Instead of defining your own function, please use `isFunctionalComponent`, `isCustomComponentElement`, and `isDOMComponentElement` which are already present in the file.